app/controllers/api/accounts_controller.rb

```diff
@@ -6,11 +6,17 @@ class Api::AccountsController < Api::BaseController
 
   represent_with Api::AccountRepresenter
 
+  private
+
   def resources
-    @accounts ||= user.try(:accounts) || Account.active.member
+    @resources ||= super.member
```

Comment:
Since the app doesn't really have any need to deal with users at this level
going forward I simplified this behavior.
On Apr 28, 2015 10:56, "Andrew Kuklewicz" <notifications@github.com> wrote:

> In app/controllers/api/accounts_controller.rb
> <https://github.com/PRX/cms.prx.org/pull/103#discussion_r29250761>:
>
> >    def resources
> > -    @accounts ||= user.try(:accounts) || Account.active.member
> > +    @resources ||= super.member
>
> This #member scope will filter out the individual account for a user,
> which previously would have been returned.
> I don't know if that is correct, but it is a change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PRX/cms.prx.org/pull/103/files#r29250761>.
>